tests/ticket/events.rb

```diff
@@ -97,22 +97,17 @@
         (result['data']['events'][4]['author']['name']).should.equal('Emilia Clarke')
         (result['data']['events'][4]['author']['staff']).should.equal(true)
 
-        (result['data']['events'][5]['type']).should.equal('DEPARTMENT_CHANGED')
```

Comment:
@guillegiu Why did you deleted this? I think it's because line 46 fails. Shouldn't you change the `departmentId` instead of removing the event test?